config/katello.migrations/160608111430-additional-foreman-proxy.rb

```diff
@@ -1,4 +1,5 @@
 def move(name, new_name=nil, default=nil)
+  return if answers['capsule'].key?(name)
```

Comment:
I don't think so... however, it probably would not hurt for me to add it there just to be safe.